npm/qsharp/src/compiler/compiler.ts

```diff
@@ -93,7 +93,12 @@ export interface ICompiler {
     operation?: IOperationInfo,
   ): Promise<CircuitData>;
 
-  getDocumentation(): Promise<IDocFile[]>;
+  getDocumentationFiles(): Promise<IDocFile[]>;
+  getDocumentationContent(
+    additionalSources?: Array<any>[],
```

Comment:
Is this really an array of arrays of any? Seems like this could have a better type.

Reply:
I changed it to [string, string][], like on other similar functions that take sources.